v22.1/cluster-setup-troubleshooting.md

```diff
@@ -463,18 +463,20 @@ CockroachDB attempts to restart nodes after they crash. Nodes that frequently re
 
 ### Decommissioning process hangs indefinitely
 
-**Explanation:** Before decommissioning a node, you need to make sure other nodes are available to take over the range replicas from the node. If no other nodes are available, the decommission process will hang indefinitely.
```

Comment:
Ah, here's the bulk of the earlier-referenced Ryan commit.

Reply:
Reverted.